cicd/jenkins/pr-centos-7-x86_64-py3-pycryptodome-pytest

```diff
@@ -7,7 +7,6 @@ runTestSuite(
     distro_version: '7',
     distro_arch: 'x86-64',
     env: env,
-    golden_images_branch: 'master',
```

Comment:
I see this removed from the Jenkins cicd stuff, and am not sure how this relates to the change occurring. Can you elaborate on this change?

Reply:
This was a change asked by @bryceml since I'm also updating the Jenkins pipeline libs version and those are no longer required.